packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -4010,70 +4023,70 @@ describe("normalizeAndValidateConfig()", () => {
 			                Please remove the field from this environment."
 		        `);
 			});
-		});
 
-		it("should error if named environment contains a `account_id` field, even if there is no top-level name", () => {
-			const rawConfig: RawConfig = {
-				legacy_env: false,
-				env: {
-					DEV: {
-						account_id: "some_account_id",
+			it("should error if named environment contains a `account_id` field, even if there is no top-level name", () => {
```

Comment:
This whole section has just been moved inside the "non-legacy" describe block as that is what they are related to. There is no code change here.